.github/workflows/masscan_container_image.yml

```diff
@@ -0,0 +1,59 @@
+name: Create and publish Masscan Docker image
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches: [ main ]
+    paths:
+      - '.github/masscan_dockerfile/Dockerfile'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - '.github/masscan_dockerfile/Dockerfile'
```

Comment:
Also add `.github/workflows/masscan_container_image.yml` so the build is also triggerd when we update the workflow.